### ISSUE TITLE
Jenkins' eye colour is not correctly set when he's in armour

### ISSUE THREAD

**Issue by AudemusN7** [henbagle/LE1CommunityPatch#153]
Credit to Murple for spotting this bug:

**Describe the bug**
In LE1, Jenkins is presented with different eye colours in every file he's in. When he is in fatigues on the Normandy his eyes are blue (NOR10_03_DS1), then when he's in the cargo bay his eyes are dark green (NOR10_11_DS1). Finally, when he's in the party, his eyes use the default eye material which is brown (Jenkins_Hench_C).

**To Reproduce**
Play through the prologue. Look at Jenkins eyes in each scene he's in (_viewmode unlit_ will give best results)

**Expected behavior**
Jenkins should obviously have a consistent eye colour throughout the prologue.

**Other Considerations**
The main issue here is figuring out which one is correct. On one hand, the scene where he's in fatigues is well lit and you can very clearly see his eyes, so it would be incredibly easy for BioWare to spot if his eyes weren't green.

However in Jenkins' hench file, if you look at his BioMorphFace, the vector for his eyes is green. This same green is also reflected in Jenkins' pawn in NOR10_11_DS1. Though it's worth considering that this data is never actually used by the game.

I think it's clear that brown is not the intended colour, but it's a toss-up between green and blue. I personally think blue would be the correct choice here, given that it's what the player is most familiar with and therefore the least disruptive version of the fix. The bright blue also fits his character better, giving him that bright/wide eyed naive look.

**Screenshots & Saves**
https://cdn.discordapp.com/attachments/778949485094240306/995467222266953808/JenkinsEyes.jpg
https://cdn.discordapp.com/attachments/778949485094240306/995477494499127306/Capture.JPG

**Game file**
>Export 778 in BIOA_NOR10_03_DS1
>Export 953 in BIOA_NOR10_11_DS1
>Export 2137 in BIOG_Jenkins_Hench_C

**Comment by henbagle** [comment]
Fixed in the cargo bay by setting `EYE_Iris_Colour_Vector` to #FF4DA1D1 in export 953 in BIOA_NOR10_11_DS1.pcc

**Comment by henbagle** [comment]
Also updated `EYE_Iris_Colour_Vector` in Export 1033 in BIOA_NOR10_11_DS1.pcc, export 872 in BIOA_NOR10_03_DS1, export 327 in BIOA_PRO10_02_DSG, and export 22 in BIOG_Jenkins_Hench_C. This fixes Jenkins' eye color whenever he's a squadmate.

**Comment by henbagle** [comment]
rondeeno fixed some more inconsistencies with Jenkin's face for the 1.7 release:

Jenkins still had some inconsistency with his appearance as well. In BIOA_NOR10_03_DS1, he uses a different BioMorphFace (BIOA_NOR_FAC.HMM.Plot.nor10_jenkins) than in his other appearances. I copied and pasted its data to the BIOG_Hench_FAC.HMM.hench_jenkins BioMorphFace in all of the files he appears in (BIOA_NOR10_03_DS1, BIOA_NOR10_11_DS1, BIOA_PRO10_02_DSG, and BIOG_Jenkins_Hench_C), and I replaced the BioMaterialInstanceConstants for his head in those files with the ones from his pawn in BIOA_NOR10_03_DS1, so he should now have the same exact values and textures in each file.
